common/metrics/config.go

```diff
@@ -202,17 +202,59 @@ var (
 		200000 * ms,
 		500000 * ms,
 		1000000 * ms,
-		2000000 * ms,
-		5000000 * ms,
-		10000000 * ms,
-		20000000 * ms,
-		50000000 * ms,
-		100000000 * ms,
-		200000000 * ms,
-		500000000 * ms,
-		1000000000 * ms,
-		2000000000 * ms,
-		5000000000 * ms,
+	}
+
+	defaultPerUnitHistogramBoundaries = map[string][]float64{
+		Dimensionless: {
+			1,
+			2,
+			5,
+			10,
+			20,
+			50,
+			100,
+			200,
+			500,
+			1000,
+		},
+		Milliseconds: {
```

Comment:
with this, do we still need to keep the defaultHistogramBoundaries?

Reply:
We should. I will add a TODO here. This is currently config in the open telemetry which I think there is an issue with it.